docs/Developers/Review.rst

```diff
@@ -140,6 +140,11 @@ For the Reviewers
    Bears require special attention during testing.
    `Testing Bears <http://api.coala.io/en/latest/Developers/Testing_Bears.html>`__
    provides a guideline for how to test bears.
+-  Check if the changes in the PR haven't been pushed from the master
```

Comment:
Usually, check is used with something which is to be kept in check, i.e. the undesirable thing. In case you are going to use `haven't been pushed`, go with `Ensure that the ...` instead of `Check if the ...`

Reply:
@shashank-b can i use `Make sure that the ...` ?

Reply:
Yeah, that would mean the same